docs/io.cozy.payslips.md

```diff
@@ -0,0 +1,64 @@
+[Table of contents](README.md#table-of-contents)
+
+# Payslips doctype
+
+## `io.cozy.payslips`
+
+- `vendor`: {string} - Vendor which issued the payslip
+- `date`: {date} - Date the payslip was emitted
+- `amount`: {number} - Amount of the payslip, __always positive__ even if it is the payslip of an employee. This amount should always be the amount which will be found in the bank operations.
+
+
+### Relationships
+
+The eventual related documents
+- `document`: {object} - The associated file. ex:
+```
+{
+  data: {
+    _id: "c43645a93831827c7ec512eac3006e51",
+    _type: "io.cozy.files"
+  }
+}
+```
+
+### Some more attributes for employee's payslip
+
+The are some more possible attributes for employee payslips.
+
+- `isEmployee`: {boolean} - Indicate if this payslip is paid to an employee of the owner of the cozy. If false, this is a payslip received by the owner.
+- `beneficiary`: {string} - Indicate the name of the employee
```

Comment:
Would it make sense to model this as a relationship, so some Pajemploi app could for example introduce a *childminder* doctype? Or is it preferable to put the relationship on the *childminder* side? Or maybe those `isEmployee` & `beneficiary` attributes are required by the *Banks* app for some purpose? (in which case it could be documented here)

Please note I'm not comfortable with document relationships yet, so feel free to correct me in case I'm wrong.

Reply:
Then maybe a single field like `bankOperationType: {"debit" | "credit"}` could be sufficient? But I may be plain wrong and it's not so important.